lib/crystalball/rspec/runner.rb

```diff
@@ -17,13 +17,23 @@ def run(args, err = $stderr, out = $stdout)
 
           Crystalball.log :info, "Crystalball starts to glow..."
           prediction = build_prediction
-
+          dry_prediction(prediction)
+          
           Crystalball.log :debug, "Prediction: #{prediction.first(5).join(' ')}#{'...' if prediction.size > 5}"
           Crystalball.log :info, "Starting RSpec."
 
           super(args + prediction, err, out)
         end
 
+        def dry_prediction (prediction)
```

Comment:
This method could be just the detection part, and could be called `dry_run?` (as well as have the tag named `--dry-run`) for consistency with similar command line tools.